modules/nf-core/coptr/index/tests/main.nf.test

```diff
@@ -32,29 +32,7 @@ nextflow_process {
 
     }
 
-    test("coptr index with multiple fastqs") {
```

Comment:
What is the reason to remove this test?

Reply:
OK, no problem. There might be some issues with tests as they are currently updating the CI workflow to use shards.